modules/core/kivent_core/managers/resource_managers.pyx

```diff
@@ -581,8 +581,9 @@ cdef class TextureManager(GameManager):
         def __get__(self):
             return [key for key in self._keys]
 
-    def load_image(self, source):
+    def load_image(self, source, mag_filter='linear'):
```

Comment:
I think i would prefer to default to None and in this case not to set the mag_filter at all.
```suggestion
    def load_image(self, source, mag_filter=None):
```